src/python/aqueduct_executor/operators/connectors/data/execute.py

```diff
@@ -142,9 +141,7 @@ def run_extract(
         assert all(
             isinstance(param_val, str) for param_val in input_vals
         ), "Parameter value must be a string."
-
-        parameters = dict(zip(spec.input_param_names, input_vals))
-        extract_params.compile(parameters)
+        extract_params.compile(input_vals)
```

Comment:
we don't need this step anymore?

Reply:
ah yeah - we don't need it cus we only need a list of input parameter values, which `input_vals` already is.